src/lib/litegraph/src/subgraph/SubgraphNode.ts

```diff
@@ -369,8 +369,14 @@ export class SubgraphNode extends LGraphNode implements BaseLGraph {
       widget: promotedWidget,
       subgraphNode: this
     })
-
-    input.widget = { name: subgraphInput.name }
+    const hasNode = (w: any): w is { node: LGraphNode } =>
```

Comment:
Might have to add an additional shape check, but I think we can avoid an explicit `any` here.

```suggestion
    const hasNode = (w: unknown): w is { node: LGraphNode } =>
```

Reply:
This is likely a missed checkout after reviewing #5024. Here, unknown is insufficient because an unknown `w` doesn't have a `.node` attribute.

Reply:
Oops, was testing Austin's PR forgot to checkout back to main before making this PR.